index.html

```diff
@@ -17,8 +16,93 @@ <h1>The Revamped Recessionary Ranch (feat. The Broke Farmer)</h1>
+        </p>
+        <p>
+          The status window to the right of the game screen can be used to track your total profit, what seed you're currently planting, how full your watering can is, and your ingredient inventory (how many of each ingredient you currently have).
+        </p>
+
+        <div id="instructions_cols">
+          <div>
+            <h3>Planting, Growing, Harvesting, Buying</h3>
+            <ul>
+              <li>
+                Press P to toggle the pop-up window that shows you what seeds you can plant and how many seeds you have for each ingredient. Click the image of the ingredient you want to plant.
+              </li>
+              <li>
+                Walk up to a plot of soil and press SPACE to plant a seed. Wait while the plant goes through 4 stages of growth - when it's fully matured and looks as it appears in the inventory or seed window, press SPACE in front of it to harvest it.
+              </li>
+              <li>
+                You can speed up the growth of your crops by watering them. Walk up to the pond and press SPACE to fill your watering can. Walk up to a plant and press SPACE to water it.
+              </li>
+              <li>
+                Plants can be watered as many times as you want until they're fully grown. However, your water is limited - you can only water plants 5 times before before the watering can is empty again!
+              </li>
+              <li>
+                If you run out of seeds for any plant, head to the marketplace in the bottom right of the farm. Walk up to the seed stall (left stall) and press SPACE in front of the crate of the ingredient you need more seeds for. Once you buy a seed for $1, your seed bag will display your updated seed total.
+              </li>
+              <li>
+                Running out of soil plots? You can purchase more in the bottom left area of the farm! Each column of 5 plots is $20. Walk up to a picket and press SPACE to purchase the column of plots above it.
+              </li>
+            </ul>
+          </div>
+
+          <div>
+            <h3>Udderly Amoosing Cow Game</h3>
+            <p>You can only obtain milk through this VR minigame!</p>
+            <ul>
+              <li>
+                Walk up to the cow surrounded by flower trees inside the fenced pen at the top right of the farm. Press SPACE next to the cow to begin the VR minigame. You'll be transported to a 3D world!
+              </li>
+              <li>
+                Use your mouse to rotate your view, and click on and collect the drops of milk falling from the sky. 5 drops makes 1 bottle of milk! You can see how much milk you've collected on the display board next to the cow.
+              </li>
+              <li>
+                Collect 10 milk bottles' worth of milk drops to end the game automatically and return to the farm. If you want to end the game early, press on the yellow box just below the display board. 
```

Comment:
don't think the game ends automatically at ten?

Reply:
it should trigger the same thing as clicking the yellow button when you collect 10!